draft-liu-nasr-architecture.md

```diff
@@ -101,38 +101,76 @@ This process is repeated periodically to continuously assure compliance.
+ 
+Relying Party (customer) then sends the Path Evidence inquiry to check and attest to this path. Following the same procedure, the client of other side would return the Path Attestation Result back, through the operators. The Operator 1 would send back a comprehensive answer/report to the Client. 
+
+Also, the operators may have heterogeneous network devices from different vendors. Since vendors provide Verifier software/hardware and Reference Values, Verifiers can be deployed either outside the operators (Fig 2) or inside of the operators (Fig 3). 
+
+                          +----------------------------------------------------------------+                                 
+                          |                                                                |                                 
+                          |       +------------+                 +------------+            |                                 
+                          |       | Verifier   |                 | Verifier   |            |                                 
+                          |       | Owner      |                 | Owner      |    ...     |                                 
+                          |       | Vendor A   |                 | Vendor B   |            |                                 
+                          |       +-------+----+                 +------+-----+   Vendors  |                                 
+                          |               |                             |                  |                                 
+                          +---------------+-----------------------------+------------------+                                 
+                                          |  Verifier software/hardware |                                                    
++-------------------+                     |  Reference Value            |                       +-------------------+        
+|                   |            +--------+----------+        +---------+---------+             |                   |        
+|                   |            |        |          |        |         |         |             |                   |        
+|    Client  A      |            |  +-----v------+   |        |   +-----v------+  |             |    Client  B      |        
+|                   |  Path      |  | Verifier   |   |        |   |  Verifier  |  | Path        |                   |        
+|  +--------------+ |  Request   |  | Vendor A   |   |        |   |  Vendor B  |  | Attestation |   +------------+  |        
```

Comment:
I think "verifier from vendor A" is better than verifier vendor A

Reply:
OK